src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/builders/impl/DynamicIncludeFlatBuilder.java

```diff
@@ -147,17 +155,41 @@ public boolean checkNotNullValue(TemplateBuilderContext context) {
      */
     public TemplateBuilder getIncludingNodeBuilder(String key) {
         if (!includingNode.isObject()) return null;
+        return getBuilderFromNode(key, includingNode);
+    }
 
+    /**
+     * @param key
+     * @param sample
+     * @return
+     */
+    public TemplateBuilder getIncludeFlatBuilder(String key, Feature sample) {
```

Comment:
I would complete the java docs here since the method is public.

Reply:
Right, doing so now.